spec/features/user_edits_a_review_spec.rb

```diff
@@ -51,4 +51,28 @@
 
     expect(page).to have_content("You must be the original author to edit!")
   end
+
+  scenario 'user edits a review and average corrects itself' do
+    user = FactoryGirl.create(:user)
+    program = FactoryGirl.create(:program)
+    FactoryGirl.create(:review, rating: 5, program: program)
+    review_2 = FactoryGirl.create(
+    :review,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.